HOMEWORK_8/EX5/RecursiveFunctions.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include "RecursiveFunctions.h"
+
+void printFromBiggerToSmallerByRecursion(int count)
+{
+    if(count < 1)
+        return;
+
+    std::cout<<count<<std::endl;
+    printFromBiggerToSmallerByRecursion(count - 1);
+}
+
+void printFromSmallerToBiggerByRecursion(int count, int number)
+{
+    if(number > count)
+    return;
+
+    std::cout<<number<<std::endl;
+    printFromSmallerToBiggerByRecursion(count, number + 1);
+}
```

Comment:
You don't need additional variables at all in any form. Neither inside of the function nor as a parameter.

Reply:
done x2